src/components/elements/slider.svelte

```diff
@@ -36,44 +36,44 @@
     })
 
     function resizeWindow() {
-        elementX = element.getBoundingClientRect().left
+        if (element) elementX = element.getBoundingClientRect().left
     }
 
     // Allows both bind:value and on:change for parent value retrieval
-    function setValue(val) {
+    function setValue(val: number) {
         value = val
         dispatch('change', {value})
     }
 
-    function onTrackEvent(e) {
+    function onTrackEvent(e: MouseEvent | TouchEvent) {
         // Update value immediately before beginning drag
         updateValueOnEvent(e)
         onDragStart(e)
     }
 
-    function onHover(e) {
-        thumbHover = thumbHover ? false : true
-    }
+    // function onHover(e) {
```

Comment:
Should we just remove this function?

Reply:
This is a useless method, I will delete it